src/phoenix/db/database.py

```diff
@@ -0,0 +1,351 @@
+import json
+import sqlite3
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Iterator, Optional, Tuple, cast
+
+import numpy as np
+from openinference.semconv.trace import SpanAttributes
+
+from phoenix.trace.schemas import ComputedValues, Span, SpanContext, SpanKind, SpanStatusCode
+
+_CONFIG = """
+PRAGMA foreign_keys = ON;
+PRAGMA journal_mode = WAL;
+PRAGMA synchronous = OFF;
+PRAGMA cache_size = -32000;
+PRAGMA busy_timeout = 10000;
+"""
+
+_INIT_DB = """
+BEGIN;
+CREATE TABLE projects (
+    rowid INTEGER PRIMARY KEY,
```

Comment:
Do index names usually not include underscores? Looks kind of weird.

Reply:
`rowid` is from sqlite by default (https://www.sqlite.org/rowidtable.html), but i had to spell it out otherwise the foreign key constraint doesn't work. We can also name it something else, but I just left it as is.